source/connection-troubleshooting.txt

```diff
@@ -204,6 +204,43 @@ Consider setting ``maxConnectionLifeTime`` and
 maintains a connection to a MongoDB instance. For more information about these
 parameters, see :ref:`Connection Pool Settings <mcs-connectionpool-settings>`.
 
+.. _java-server-selection-errors:
+
+Server Selection Errors
+-----------------------
+
+MongoDB might not be able to complete a request even when some servers
+are available, causing the driver to return a server selection error.
+The following list describes some situations in which you
+might encounter a server selection error:
+
+- You use tag sets in a read preference specification when no server
+  with those tags exists.
+- You write to a replica set when the primary node is unavailable.
+
+This error is of type ``MongoTimeoutException``. The following code
```

Comment:
```suggestion
This exception is of type ``MongoTimeoutException``. The following
```

Reply:
Because it's not code, it's the string representation of the exception that is likely to appear in your log file